float32 speed
float32 current
float32 motor_temp
float32 controller_temp
float32 controller_volt
float32 water
